extensions/big-data-cluster/src/bigDataCluster/constants.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+
+export enum BdcItemType {
+	controllerRoot = 'bigDataClusters.itemType.ControllerRootNode',
+	controller = 'bigDataClusters.itemType.ControllerNode',
+	folder = 'bigDataClusters.itemType.FolderNode',
+	sqlMaster = 'bigDataClusters.itemType.SqlMasterNode',
+	EndPoint = 'bigDataClusters.itemType.EndPointNode',
+	addController = 'bigDataClusters.itemType.AddControllerNode',
+}
+
+export class IconPath {
+	private static extensionContext: vscode.ExtensionContext;
+
+	public static controllerNode: { dark: string, light: string };
+	public static folderNode: { dark: string, light: string };
+	public static sqlMasterNode: { dark: string, light: string };
+
+	public static setExtensionContext(extensionContext: vscode.ExtensionContext) {
+		IconPath.extensionContext = extensionContext;
+		IconPath.controllerNode = {
+			dark: IconPath.extensionContext.asAbsolutePath('resources/light/bigDataCluster_controller.svg'),
```

Comment:
>bigDataCluster_controller [](start = 67, length = 25)

You should get a dark-mode theme icon for this

Reply:
This iconpath is just copied from CMS. It share the same icon for dark and light.

Reply:
I don't think CMS would have a bigDataCluster_controller.svg icon...and anyways if you don't have a dark mode version then you should get that from design.

Reply:
I copied them over, and renamed the icons. The icons will be reviewed by the design team tomorrow, and I will change the icons if requested.
Here is existing code convention in CMS:
https://github.com/microsoft/azuredatastudio/blob/ecef90dc8b1066ee18dd568d7f959a44aa300a1d/extensions/cms/src/cmsResource/tree/registeredServerTreeNode.ts#L50-L62

Reply:
Gene - this is a bug in CMS.
- Can you open a bug against this and get Aditya to fix for CMS?
- even if this is a placeholder, can you check in pointing to the dark folder path and then update the icon in a separate PR?

Reply:
fixed.